django/santropolFeast/member/models.py

```diff
@@ -174,7 +187,7 @@ class Meta:
 
     emergency_contact = models.ForeignKey(
         'member.Member',
-        verbose_name=_('emergency_contact'),
+        verbose_name=_('Emergency Contact'),
```

Comment:
Be carefull we do not write in English but we just create id which will then be translated.